x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.tsx

```diff
@@ -154,7 +154,7 @@ const EventDetailsPanelComponent: React.FC<EventDetailsPanelProps> = ({
 
   return isFlyoutView ? (
     <>
-      <EuiFlyoutHeader hasBorder>
+      <EuiFlyoutHeader hasBorder={isHostIsolationPanelOpen ? true : false}>
```

Comment:
you can just pass `hasBorder={isHostIsolationPanelOpen}` assuming `isHostIsolationPanelOpen` is a boolean? Otherwise you can `!!isHostIsolationPanelOpen`

Reply:
yeah, I can just put isHostIsolationPanelOpen 👍

Reply:
Sweet, thanks!